MAINTAINERS_GUIDE.md

```diff
@@ -0,0 +1,90 @@
+
+This document is a manual for maintainers old and new.
+It explains what is expected of maintainers, how they should work, and what tools are available to them.
+
+This is a living document - if you see something out of date or missing, speak up!
+
+## What are a maintainer's responsibility?
+
+It is every maintainer's responsibility to:
+
+1) Expose a clear roadmap for improvements for the project.
+2) Deliver prompt feedback and decisions on pull requests.
+3) Be available to anyone with questions, bug reports, criticism etc. on their component.
+   This includes Slack, mailing-list, and GitHub issues and pull requests.
+4) Make sure to respect the philosophy, design, compatibility considerations, and roadmap of the project.
+
+## How are decisions made?
+
+Short answer: with pull requests to this repository.
+
+This is an open-source project with an open design philosophy.
+This means that the repository is the source of truth for EVERY aspect of the project, including its philosophy, design, roadmap and APIs.
+*If it's part of the project, it's in the repo.
+It's in the repo, it's part of the project.*
+
+As a result, all decisions can be expressed as changes to the repository.
+A spec change is a change to the specification.
+An implementation change is a change to the source code.
+A philosophy change is a change to the philosophy manifesto.
+And so on.
+
+All decisions affecting this project, big and small, follow the same 3 steps:
+
+* Step 1: Open a pull request. Anyone can do this.
+
+* Step 2: Discuss the pull request. Anyone can do this.
+
+* Step 3: Accept (`LGTM`) or refuse a pull request. The relevant maintainers do this (see below "Who decides what?")
+
+*I'm a maintainer, should I make pull requests too?*
+
+Yes.
+Nobody should ever push to the `main` branch directly.
+All changes should be made through a pull request.
+
+Bigger changes (that may span projects): an [OCI Working Group](https://github.com/opencontainers/tob/blob/main/CHARTER.md) may be needed.
+
+## Who decides what?
+
+All decisions are pull requests, and the relevant maintainers make decisions by accepting or refusing the pull request.
+Review and acceptance by anyone is denoted by adding a comment in the pull request: `LGTM`.
+However, only currently listed `MAINTAINERS` are counted towards the required **two LGTMs**.
+
+Overall the maintainer system works because of mutual respect across the maintainers of the project.
+The maintainers trust one another to make decisions in the best interests of the project.
+Sometimes maintainers can disagree and this is part of a healthy project to represent the point of views of various people.
+
+The maintainer system is built on trust, if there is a conflict the maintainers do not agree on it can be brought to the [technical oversight board](https://github.com/opencontainers/tob) for resolution.
```

Comment:
```suggestion
The maintainer system is built on trust.
If there is a conflict the maintainers do not agree on it can be brought to the [technical oversight board](https://github.com/opencontainers/tob) for resolution.
```